spring-auto-restdocs-core/src/test/java/capital/scalable/restdocs/payload/JacksonModelAttributeSnippetTest.java

```diff
@@ -260,6 +262,31 @@ public void accessors() throws Exception {
                         .row("bothWays", "String", "true", ""));
     }
 
+    @Test
+    public void withoutAnnotation() throws Exception {
+        HandlerMethod handlerMethod = createHandlerMethod("withoutAnnotation", Item.class, ParentItem.class,
+                DeprecatedItem.class);
+
+        HandlerMethodArgumentResolver modelAttributeMethodProcessor = new ServletModelAttributeMethodProcessor(true);
+        new JacksonModelAttributeSnippet(singletonList(modelAttributeMethodProcessor), false).document(operationBuilder
+                .attribute(HandlerMethod.class.getName(), handlerMethod)
+                .attribute(ObjectMapper.class.getName(), mapper)
+                .attribute(JavadocReader.class.getName(), javadocReader)
+                .attribute(ConstraintReader.class.getName(), constraintReader)
+                .build());
+
+        assertThat(this.generatedSnippets.snippet(AUTO_MODELATTRIBUTE)).is(
+                tableWithHeader("Parameter", "Type", "Optional", "Description")
+                        .row("field1", "String", "true", "")
```

Comment:
This field in `Item` is annotated as `@NotBlank`. I'm not sure these annotations work for ModelAttribute.

Reply:
I'll have a look at this later today.

Reply:
The detection of the ModelAttribute without an annotation didn't work. I implemented that ModelAttribute detection according to what you've stated in #426:

Any other argument | If a method argument is not matched to any of the earlier values in this table and it is a simple type (as determined by BeanUtils#isSimpleProperty, it is a resolved as a @RequestParam. Otherwise, it is resolved as a @ModelAttribute.
-- | --